docs/rest_api.rst

```diff
@@ -200,23 +200,81 @@ To delete a link:
     DELETE /course_access_groups/api/v1/group-courses/2/
 
 
-Marking Courses as Public
+Setting Courses as Public
 -------------------------
+When the course access group feature is enabled, by default all courses are
+private and only accessible to learners with memberships to groups that
+has those courses.
 
-These endpoints allow mark courses as a public.
-This means the course is available to
-all site learners regardless of their Course Access Groups membership.
+However, some users could have no group membership so they won't have access
+to the private courses. To a course available to those learners and everyone
+else a course should be made public.
+
+This endpoint allow making a course a public one.
```

Comment:
"This endpoint allows" if we have a single endpoint. "These endpoints allow" if we have more than one endpoint.

Question, does the endpoint "allow" or does the endpoint actually do the state change? I'm assuming the later, so how about "This endpoint sets the course as public"?

Reply:
Noted and amended. Thanks @johnbaldwin! 

Please keep those coming. Writing docs is difficult enough. It's even more difficult I'm translating from Arabic in my head.